angular.json

```diff
@@ -132,7 +132,10 @@
               "apps/geomat/tsconfig.app.json",
               "apps/geomat/tsconfig.spec.json"
             ],
-            "exclude": ["**/node_modules/**", "!apps/geomat/**/*"]
```

Comment:
Unnecessary formatting please use `nx format:check --base=origin/dev` to check for wrong formatted files. 
`nx format:write --base=origin/dev` can fix wrong formatted files.